requirements/dev.txt

```diff
@@ -7,6 +7,10 @@ pytest==6.2.2
 pytest-asyncio==0.15.1
 pytest-mock==3.6.1
 tox==3.24.1
+# for mlflow tests
+torch==1.9.0
```

Comment:
Since this is only required for the `mlserver-mlflow` package, should we remove them from this one?

Reply:
github tests only install from this list and it doesnt install the dev requirements for `mlserver-mlflow`

Reply:
Mmmh weird. Doesn't it run `make install-dev`? That one should install any `requirements-dev.txt` present on the runtime packages.

Reply:
it does yes and it should work, so removing these packages from here.